.travis.yml

```diff
@@ -14,7 +14,7 @@ node_js:
   - '4'
   - '5'
 before_install:
-  - npm i -g npm@^3.0.0
+  - if [[ `node -v | sed 's/[^0-9\.]//g'` < 4 ]]; then npm i -g npm@^2.0.0; else npm i -g npm@^3.0.0; fi
```

Comment:
Is it possible to comment in `.yml`? If so, let's add a comment about what on earth this is doing :-)